lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -821,11 +859,11 @@ def add_threeds_2_ucaf_data(xml, payment_method, options)
       def stored_credential_commerce_indicator(options)
         return unless options[:stored_credential]
 
-        return if options[:stored_credential][:initial_transaction]
-
         case options[:stored_credential][:reason_type]
-        when 'installment' then 'install'
```

Comment:
I don't see anywhere in the docs indicating the keyword has changed from `install` to `installment`

Reply:
That's what I got back from the CyberSource contact. I'll forward you the email

Reply:
ugh i hate when what they say does not match their docs.

Reply:
I emailed them again just to confirm because like you said it says install in docs. I'll keep you updated.

Reply:
@aenand It's actually ` install` but I've decided to remove stored credentials from here and update it in a separate ticket to keep this as simple as possible.